python/ray/air/tests/execution/test_tracked_actor.py

```diff
@@ -0,0 +1,231 @@
+from collections import Counter
+from typing import Any, Optional, Type
+
+import pytest
+
+import ray
+from ray.air import ResourceRequest
+from ray.air.execution import FixedResourceManager, PlacementGroupResourceManager
+from ray.air.execution._internal.event_manager import RayEventManager, EventType
+
+
+def _raise(exception_type: Type[Exception] = RuntimeError, msg: Optional[str] = None):
+    def _raise_exception(*args, **kwargs):
+        raise exception_type(msg)
+
+    return _raise_exception
+
+
+class Started(RuntimeError):
+    pass
+
+
+class Stopped(RuntimeError):
+    pass
+
+
+class Failed(RuntimeError):
+    pass
+
+
+class Result(RuntimeError):
```

Comment:
all events are RuntimeErrors?
Started, Stopped, and Result are not errors :)

Reply:
But it's easy to catch :-) It's just an event we make sure is triggered